### PR TITLE
Modal feature branch

### PR DESCRIPTION
**Pull request by joeveloper**
The following features were implemented

Modal popup window on click of the project buttons
The modal close feature
Responsive view adjusted

### CODE REVIEW

**Comment by huseyinbiyik** [comment]
Hi @joeveloper 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

Alt Text

## Invalid Code Review Request

You have submitted a project, that has templated text in README file. Please remove these texts in section aiming to inform you.
Screenshot 2023-03-09 at 10 14 01 PM


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by nahnahdev — COMMENTED** [review]
## Project Status: INVALIDATED :red_circle: 
Hi @joeveloper , 

While you made a great effort in this project, I can't go ahead and review your code.


## Invalid Code Review Request

- This is because you have submitted a project, that is a pair programming activity and hence need to have commits from your partner/partners to show that you all worked together on this project :wink: 
- Your commit history
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @NahnahAJ in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by joeveloper** [comment]
All review comments have been treated.

File: index.js [hunk 1]
```diff
@@ -86,7 +86,8 @@ const popupMenu = [
   },
 ];
 
-const test = popupMenu.map((item) => `
+const test = popupMenu.map(
+  (item) => `
    <button rel='noopener noreferrer' href='javascript:void(0)' id='close-modal'>&times;</button>
   <div class='modal-heading'>
         <h2>${item.title}</h2>
```
Comment (Qoosim) [inline index.js:93]: - [x] Great job so far, however, there're few things you need to correct before your project could be approved.

-    You're required to remove already created HTML code for projects list in the `index.html`, and instead it should be added via JavaScript. and all of the HTML in that section is created dynamically when the page loads. No code for projects list should remain in the `index.html` again and instead should be created dynamically from your javascript file when the page is loaded.
- The popup window, both the desktop and the mobile must stick to the figma design. Kindly check the two screenshots of your design and the figma and work your alongside the figma design. At  this mement, all the information on your popup window is not visible on the screen when it opens. Kindly look into that  and fix.
### Project requirement
project-rqt


### Your Design Desktop
[your-desktop-popup.webm](https://user-images.githubusercontent.com/26597432/224278084-72d598ec-71b7-49a8-8c3f-c35aed27c250.webm)

### Figma Design Desktop
temp-desktop-popup


### Your Design Mobile
[your-mobile-popup.webm](https://user-images.githubusercontent.com/26597432/224278189-722a9ad1-328e-480b-80cf-eedcb3d8e837.webm)

### Figma Design Mobile
temp-mobile-popup

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (joeveloper) [reply index.js:93]: resolved

**Review by AmaduKamara — APPROVED** [review]
# Status: Approved 🟢 🟢 🟢

Hi @joeveloper,

Good job working on the changes highlighted by the previous reviewers 👍🏽

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Highlights:

✅ You used Gitflow correctly 👍🏽
✅ Linter checks passed 👍🏽
✅ Modal data is dynamically fetched and populated 👍🏽
✅ Modal is responsive as expected 👍🏽

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**